pubspec.yaml

```diff
@@ -24,8 +24,13 @@ dependencies:
   rudder_integration_appsflyer_flutter: 1.1.9
   rudder_integration_adjust_flutter: 1.1.4
   rudder_integration_leanplum_flutter: 1.1.4
+  rudder_integration_kochava_flutter: 1.0.0
   flutter_plugin_tools: ^0.12.1
 
+dependency_overrides:
```

Comment:
why is this needed?

Reply:
added this while testing and it is not needed, removed it